x-pack/legacy/plugins/siem/public/components/ml/tables/get_anomalies_host_table_columns.tsx

```diff
@@ -48,16 +54,25 @@ export const getAnomaliesHostTableColumns = (
     field: 'anomaly.severity',
     sortable: true,
     render: (_, anomaliesByHost) => (
-      <AnomalyScore
-        startDate={startDate}
-        endDate={endDate}
-        jobKey={`anomalies-host-table-severity-${createCompoundHostKey(anomaliesByHost)}`}
-        narrowDateRange={narrowDateRange}
-        interval={interval}
+      <DraggableScore
+        id={`anomalies-host-table-severity-${createCompoundHostKey(anomaliesByHost)}`}
         score={anomaliesByHost.anomaly}
       />
     ),
   },
+  {
+    name: i18n.DETECTOR,
```

Comment:
Would it be possible to move the `Job Name` column just before the `Anomaly Score` column in all instances? In thinking about it, such context before the score might be useful.